TYPO3.Neos/Classes/TYPO3/Neos/Service/View/NodeView.php

```diff
@@ -237,7 +237,9 @@ public function collectParentNodeData(NodeInterface $rootNode, array $nodes)
                     $collectTreeNodeData($children, $childNode);
                 }
             }
-            $treeNodes[] = $self->collectTreeNodeData($node['node'], true, $children, $children !== array(), isset($node['matched']));
+            if (isset($node['node']) && $node['node'] instanceof NodeInterface) {
```

Comment:
I don't understand how this happen, on line 208, we call getPath method, so the node variable should be a NodeInterface instance. But in my case it does not throw an exception here, but only on line 240